napari/_qt/qt_event_loop.py

```diff
@@ -132,6 +133,17 @@ def get_app(
         app.setWindowIcon(QIcon(kwargs.get('icon')))
         set_app_id(kwargs.get('app_id'))
 
+        if os.getenv("NAPARI_CATCH_ERRORS") not in ('0', 'False'):
```

Comment:
Also, I think I'd rather just _always_ connect the `NapariQtNotification.show_notification` message and then handle whether or not to display the toast in `NapariQtNotification` itself.  With this approach here, once it fails to be connected when the app starts up, we can never go back and decide later that we _would_ like to see messages?  (which would be easier if it were just a property on the `NapariQtNotification``

Reply:
will handle this in a later PR